### PR TITLE
Add containerd installation on Debian

### PR DESCRIPTION
**Pull request by peymanslh**
Add `containerd` installation for Debian 9+ based on [docker documentation](https://docs.docker.com/engine/install/debian/).

### CODE REVIEW

**Comment by peymanslh** [comment]
CLA signed!

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -143,6 +143,44 @@ sudo mkdir -p /etc/containerd
...
+```shell
+# (Install containerd)
+## Set up the repository
+### Install packages to allow apt to use a repository over HTTPS
+sudo apt-get update && sudo apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+```
+
+```shell
+## Add Docker's official GPG key
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:162]:
```suggestion
curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key --keyring /etc/apt/trusted.gpg.d/docker.gpg add -
```

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -143,6 +143,44 @@ sudo mkdir -p /etc/containerd
...
+   stable"
+```
+
+```shell
+## Install containerd
+sudo apt-get update && sudo apt-get install -y containerd.io
+```
+
+```shell
+# Configure containerd
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:179]:
```suggestion
# Set default containerd configuration
```

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -143,6 +143,44 @@ sudo mkdir -p /etc/containerd
...
+sudo mkdir -p /etc/containerd
+sudo containerd config default | sudo tee /etc/containerd/config.toml
```
Comment (sftim) [inline content/en/docs/setup/production-environment/container-runtimes.md:181]: Would:
```suggestion
containerd config default | sudo tee /etc/containerd/config.toml
```
also work?

**Review by sftim — COMMENTED** [review]
Hi @peymanslh

I would write the Docker key to its own keyring inside `/etc/apt/trusted.gpg.d`, as this is easier to revert if needed. The other two feedback items are less important.

**Comment by bart0sh** [comment]
/lgtm
@peymanslh  It would be nice to squash all 4 commits into one

**Comment by peymanslh** [comment]
@bart0sh @sftim 
Is there anything else that I need to do?

**Review by sftim — COMMENTED** [review]
/approve
Thanks!

**Pull request merged**